bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/coap/ShellyCoapJSonDTO.java

```diff
@@ -223,9 +223,17 @@ public CoIotDevDescription read(final JsonReader in) throws IOException {
                 }
 
                 in.endArray();
+                if (in.hasNext()) {
+                    name = in.nextName();
+                }
             }
-            in.endObject();
 
+            if (name.equalsIgnoreCase(COIOT_TAG_ACT)) {
+                // skip record
+                in.skipValue();
+            }
```

Comment:
Looks like you did not based your change on the most uptodate main because this part is already in the there, it was merged yesterday.
Please rebase.